README.md

```diff
@@ -3,7 +3,61 @@
 It's an initial Java project to connect to Blibli Seller API. 
 Please feel free to relay your comments, suggestions or corrections through pull request.
 
-Installation step:
```

Comment:
Option 1: Import the final .jar into your java project.
You can copy the final .jar into your java project by downloading the final .jar below
"https://github.com/bliblidotcom/seller-api-client-java/releases/tag/1.0.0-1" //add the latest .jar link